src/components/toolbar/index.tsx

```diff
@@ -79,7 +79,7 @@ export const Toolbar = () => {
         <ToolButton data-tool='line' active={tool === 'line'}>
           <ArrowDownRightIcon />
         </ToolButton>
-        <InputColor type='color' name='color' onChange={handleChangeFillStyle} />
+        <ColorInput type='color' name='color' onChange={handleChangeFillStyle} />
```

Comment:
color input should always has type `color`, could you set this property inside this component?